webapp/channels/src/components/toast_wrapper/toast_wrapper.test.tsx

```diff
@@ -613,4 +618,105 @@ describe('components/ToastWrapper', () => {
             expect(dismissHandler).toHaveBeenCalled();
         });
     });
+
+    describe('Scroll-to-bottom toast', () => {
```

Comment:
This is regarding all the new tests.

Not really needed, but I would love we start moving away from "shallow testing" and start using the approach of react testing library (https://testing-library.com/docs/react-testing-library/intro/)

If you want to try, always appreciated, but we can leave it like this if you prefer.

Reply:
Let me check on that. Thank you.

Reply:
@larkox Just migrated to `react-testing-library`. But I'm not sure if I do it right. Hope to hear your feedback.